source/queue_policy/policies.rst

```diff
@@ -489,7 +491,7 @@ want to manage multiple packages, please request multiple /contrib
 package. You can do this all at one time when submitting your request
 to the Help System.
 
-Maintaining "Metadata" for the contrib Package
+Maintaining "Metadata" for the Contrib Package
```

Comment:
Probably should be
Maintaining "Metadata" for ``contrib`` packages
because it's not just one contrib package, and it's an explicit filesystem location (/contrib) as well.

Reply:
changed per your request as I understand it. Tell me if I've got it wrong.